test-data/unit/pythoneval.test

```diff
@@ -809,6 +809,40 @@ _program.py:3: note: Possible overload variants:
 _program.py:3: note:     def __add__(self, Tuple[str, ...]) -> Tuple[str, ...]
 _program.py:3: note:     def __add__(self, Tuple[Any, ...]) -> Tuple[Any, ...]
 
+[case testMultiplyTupleByIntegerLiteral]
+from typing import Tuple
+t = ('',) * 2
+reveal_type(t)
+t2 = ('',) * -1
+reveal_type(t2)
+t3 = ('', 1) * 2
+reveal_type(t3)
+def f() -> Tuple[str, ...]:
+    return ('', )
+reveal_type(f() * 2)
+[out]
+_testMultiplyTupleByIntegerLiteral.py:3: note: Revealed type is "Tuple[builtins.str, builtins.str]"
+_testMultiplyTupleByIntegerLiteral.py:5: note: Revealed type is "Tuple[]"
+_testMultiplyTupleByIntegerLiteral.py:7: note: Revealed type is "Tuple[builtins.str, builtins.int, builtins.str, builtins.int]"
+_testMultiplyTupleByIntegerLiteral.py:10: note: Revealed type is "builtins.tuple[builtins.str*]"
```

Comment:
Can you move these tests to `test-data/unit/check-tuples.test`? You may need to add some more definitions to to `test-data/unit/fixtures/tuple.pyi` (e.g. `__mul__`), and use this in your test cases to use the fixture:

```
[case test...]
...
[builtins fixtures/tuple.pyi]
```